test_apps/source-app/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
suggest to use `tests` as the top level folder, and then put the `source-app` folder inside it.